core/components/header/index.tsx

```diff
@@ -52,60 +58,62 @@ export const Header = async ({ cart, data }: Props) => {
 
   const locale = await getLocale();
-  const messages = await getMessages({ locale });
 
   /**  To prevent the navigation menu from overflowing, we limit the number of categories to 6.
    To show a full list of categories, modify the `slice` method to remove the limit.
    Will require modification of navigation menu styles to accommodate the additional categories.
    */
   const categoryTree = data.categoryTree.slice(0, 6);
-  const logo = data.settings && <StoreLogo data={data.settings} />;
 
   return (
-    <ComponentsHeader links={categoryTree} logo={logo}>
-      <QuickSearch logo={logo} />
-
-      {customerId ? (
-        <Dropdown
-          items={[
-            { path: '/account', name: 'My account' },
-            { path: '/account/orders', name: 'Orders' },
-            { path: '/account/messages', name: 'Messages' },
-            { path: '/account/addresses', name: 'Addresses' },
-            { path: '/account/wishlists', name: 'Wish lists' },
-            { path: '/account/recently-viewed', name: 'Recently viewed' },
-            { path: '/account/settings', name: 'Account settings' },
-            { action: logout, name: 'Log out' },
-          ]}
-          trigger={
-            <Button
-              aria-label="Account"
-              className="p-3 text-black hover:bg-transparent hover:text-primary"
-              variant="subtle"
-            >
-              <User aria-hidden="true" />
-            </Button>
-          }
-        />
-      ) : (
-        <Link aria-label="Login" className="block p-3" href="/login">
-          <User />
-        </Link>
-      )}
-      <p role="status">
-        <Suspense
-          fallback={
-            <CartLink>
-              <ShoppingCart aria-label="cart" />
-            </CartLink>
-          }
-        >
-          {cart}
-        </Suspense>
-      </p>
-      <NextIntlClientProvider locale={locale} messages={{ Header: messages.Header ?? {} }}>
-        <LocaleSwitcher />
-      </NextIntlClientProvider>
-    </ComponentsHeader>
+    <ComponentsHeader
+      account={
+        customerId ? (
+          <Dropdown
+            items={[
+              { href: '/account', label: 'My account' },
+              { href: '/account/orders', label: 'Orders' },
+              { href: '/account/messages', label: 'Messages' },
+              { href: '/account/addresses', label: 'Addresses' },
+              { href: '/account/wishlists', label: 'Wish lists' },
+              { href: '/account/recently-viewed', label: 'Recently viewed' },
+              { href: '/account/settings', label: 'Account settings' },
+              { action: logout, name: 'Log out' },
+            ]}
+            trigger={
+              <Button
+                aria-label="Account"
+                className="p-3 text-black hover:bg-transparent hover:text-primary"
+                variant="subtle"
+              >
+                <User aria-hidden="true" />
+              </Button>
```

Comment:
aria attributes should be used as a last resort
```suggestion
              <Button
                className="p-3 text-black hover:bg-transparent hover:text-primary"
                variant="subtle"
              >
                <User>
                  <title>Account</title>
                </User>
              </Button>
```